### DISCUSSION TITLE
Improvements to pull payment,  password and qr-scan address.

### DISCUSSION THREAD

**Discussion by nostitos** [btcpayserver/btcpayserver#5242] [Ideas / Feature Requests]
* Add a way to use camera to scan a qr-code to capture a receive address.

I can imagine someone is getting a refund / pull payment link on his laptop but want to receive the BTC on a wallet on his mobile.

* Add a password to a pull payment request.

The pass / PIN could be shared via a secondary communication channel to increase security.

**Comment by Yappadappa** [comment]
Receive payment

**Reply by Yappadappa** [reply]
Help
